### ISSUE TITLE
definitions and english terms for information attribute?

### ISSUE THREAD

**Issue by ElisabethDeVries** [Stichting-CROW/verkeersborden#347]
Were in the beheerplatform do we put those?

**Comment by gulaycanbaloglu** [comment]
We can write the description of the attribute in the top level concept's corresponding attribute, and give the real value in the attributes of below concepts. Example:
Icon -- has attribute --> IconPositionNumber : Definition of the what icon position number is (NL & EN)
A0110 -- has attribute --> IconPositionNumber : 1a
![image](https://github.com/user-attachments/assets/ba356f8e-d0dd-4eb8-9c33-90796058d6ff)

**Comment by gulaycanbaloglu** [comment]
Also, kleur and ral-kleur attributes for shield: we thought it is a better practice to model these two as aspects. In this way, we can easily add the dutch and english definitions and the values in qualification groups.

**Comment by ElisabethDeVries** [comment]
> Also, kleur and ral-kleur attributes for shield: we thought it is a better practice to model these two as aspects. In this way, we can easily add the dutch and english definitions and the values in qualification groups.

Yes, I saw - I deited this, becuase in de definition of the RAL-color, the color is described, so actually we only need RAL-Color.
